js/webui.js

```diff
@@ -828,10 +828,16 @@ var theWebUI =
 							}
 							case "webui.speedgraph.max_seconds":
 							{
+								theWebUI.settings["webui.speedgraph.max_seconds"] = nv;
 								theWebUI.speedGraph.setMaxSeconds(parseInt(theWebUI.settings['webui.speedgraph.max_seconds']))
 								theWebUI.speedGraph.draw();
 								break;
 							}
+							case "webui.show_open_status":
+							{
+								if (nv && theWebUI.systemInfo.rTorrent.iVersion < 0x907)
+									noty('Showing open status requires rtorrent >= v0.9.7!', 'error');
```

Comment:
Perhaps it would be better to disable the checkbox on the WebUI if the rTorrent version is less than 0.9.7. It should be possible to display a hint box "Requires rTorrent 0.9.7" when the user hovers over the disabled checkbox.

Reply:
Yeah, that's probably better
now shows disabled option with "Requires rTorrent >= v0.9.7" tooltip